### PR TITLE
Fix the warnings in tests

### PR DESCRIPTION
**Pull request by aliskin**
## Description
Fix the warnings in the tests as follows:
- Replace DataFrame.append with pandas.concat (append is deprecated);
- Make a copy of DataFrame before assigning;
- Explicitly create an empty Series and specify dtype when the Series is None;
- Catch the user warning when it's intended.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation and examples improvement (changes affected documentation and/or examples)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: crowdkit/aggregation/classification/dawid_skene.py [hunk 1]
```diff
@@ -267,9 +267,10 @@ def _m_step(data: annotations.LABELED_DATA, probas: annotations.TASKS_LABEL_PROB
         worker's errors probabilities matrix.
         """
         skilled_data = data.copy()
-        skilled_data['skill'] = probas.lookup(data.task, data.label)
+        skilled_data['skill'] = pd.Series(
+            [probas.loc[row.task, row.label]
+                for _, row in data.iterrows()], dtype=np.float64)
```
Comment (pilot7747) [inline crowdkit/aggregation/classification/dawid_skene.py:272]: Seems like it's gonna dramatically worsen the performance. `.iterrows()` is extremely slow.

**Review by pilot7747 — CHANGES_REQUESTED** [review]
Please check if there are changes in computation performance on relevance-5 and crowdspeech-test-clean.

File: crowdkit/aggregation/classification/dawid_skene.py (hunk 1 above)
Comment (aliskin) [reply crowdkit/aggregation/classification/dawid_skene.py]: As we discussed, I checked the performance and indeed, iterrows() is very slow. In the end, I replaced lookup method with a solution recommended in [pandas documentation](https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#looking-up-values-by-index-column-labels).
Tests on relevance-2 and relevance-5 datasets showed that the performance is currently almost the same.

**Review by pilot7747 — APPROVED** [review]
Thank you!

**Pull request merged**